src/components/results/body/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { TREE_VIEW, JSON_VIEW } from '../results-view-selector';
+import JSONTree from 'react-json-tree';
+import { escapeLikeJSON, stringify } from '../utils';
+
+import './style.css';
+
+class RequestBody extends React.Component {
```

Comment:
given that this is nothing more than a `render` function and that `RequestHeader` itself is just a function, is there a reason not to make `RequestBody` a single function also?